lang_portal_backend/api/sentence_builder_views.py

```
@@ -0,0 +1,216 @@
+)
+
+# Initialize ChromaDB client
+chroma_client = chromadb.Client(chromadb.Settings(
+    persist_directory=CHROMA_PERSIST_DIR,
+    chroma_db_impl="duckdb+parquet",
+))
+
+# Create or get the collection
+try:
+    sentence_collection = chroma_client.get_collection("sentence_patterns")
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)